server/src/main/java/io/deephaven/server/plugin/js/JsPluginModule.java

```diff
@@ -54,10 +53,7 @@ static Set<Registration> providesConfigDirRegistration() {
     @Provides
     @ElementsIntoSet
     static Set<Registration> providesPackageRoots() {
-        return jsPluginsPackageRoots()
-                .stream()
-                .map(Registration.class::cast)
-                .collect(Collectors.toSet());
+        return Set.copyOf(jsPluginsPackageRoots());
```

Comment:
why not apply to the other unnecessary streams above too?

Reply:
The above aren't streams, but `Optional`s that turn into `Set.of(Element)` or `Set.of()`.

Reply:
providesConfigDirRegistration and providesResourceBaseRegistration, the two methods above this?

Reply:
Ah got it, thanks for clarifying.